parsec/core/gui/lang.py

```diff
@@ -20,13 +19,14 @@
 
 
 def format_datetime(dt, full=False, seconds=False):
-    fmt = "L LT"
-    if seconds:
-        fmt = "L LTS"
-    if full:
-        fmt = "LLLL"
-    # The alternative formatter is now the default one since pendulum 2.0.0
-    return dt.in_tz(pendulum.local_timezone()).format(fmt, locale=_current_locale_language)
+    if _current_locale_language == "fr":
+        if seconds:
+            return dt.format("%d/%m/%Y %H:%M:S")
+        return dt.format("%d/%m/%Y %H:%M")
+    else:
+        if seconds:
+            return dt.format("%m/%d/%Y %I:%M:S%p")
+        return dt.format("%m/%d/%Y %I:%M%p")
```

Comment:
Can you use [`QLocal::dateTimeFormat`](https://doc.qt.io/qt-6/qlocale.html#dateTimeFormat) ?

[`PyQt5 QLocale::dateTimeFormat()`](https://www.riverbankcomputing.com/static/Docs/PyQt5/api/qtcore/qlocale.html#dateTimeFormat)